packages/main/src/plugin/image-registry.ts

```diff
@@ -758,12 +771,23 @@ export class ImageRegistry {
     return response.token;
   }
 
-  async doCheckCredentials(scheme: string, authUrl: string, username: string, password: string): Promise<void> {
+  async doCheckCredentials(
+    scheme: string,
+    authUrl: string,
+    username: string,
+    password: string,
+    ignoreInvalidCert?: boolean,
+  ): Promise<void> {
+    const options = this.getOptions();
+    // If ignoreInvalidCertificate was passed in, ignore the certificate
+    if (ignoreInvalidCert && options.https) {
+      options.https.rejectUnauthorized = false;
+    }
```

Comment:
as we have tthe getOptions() that setup the options, I would add the parameter there

getOptions(ignoreInvalidCert)

because at two different places we're doing
if (ignoreInvalidCert && options.https) {
      options.https.rejectUnauthorized = false;
}